pages/logout.page.js

```diff
@@ -0,0 +1,23 @@
+import BasePage from '../pages/base.page';
+
+class LogoutPage extends BasePage{
+
+    //page locators:
+    get userMenu(){return $('.user-avatar')}
+    get logoutOption(){return $('=Logout')}
+    get logoutSuccess(){return $('.badge')}
+
+
+    //page actions:
+    async logout(){
+
+        await this.userMenu.click();
+        await this.logoutOption.click();
+
+        await expect(this.logoutSuccess).toHaveTextContaining('Success');
```

Comment:
Assertions belong conceptually to the test, not the page object. The page object is only concerned with modeling page elements and behaviour, but it's not assigned the responsibility to evaluate the correctness of a result, as that is something the test knows, and it's also a concept relative to the specific test case implementing it.

Remove this last line from this method, and put it directly in the test case that asserts logout behaviour instead.